examples/api/test/widgets/basic/indexed_stack.0_test.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/basic/indexed_stack.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('has correct forward rendering mechanism',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.MyApp(),
+    );
+
+    final Finder gesture2 = find.byKey(const Key('gesture2'));
+    final Element containerFinder =
```

Comment:
This file looks like you're using `dart format` for formatting these, and in the framework code we don't use that.

Check out [this part of the style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting).

That said, I find that a good starting point for formatting is to apply `dart format --output=write --line-length=120 <file>`  (you can also just set the line length in the IDE prefs), and then fix up by hand anything that still looks wonky. Turn off "Format on Save" in the IDE prefs.